src/Development/IDE/Core/FileStore.hs

```diff
@@ -198,20 +189,20 @@ ideTryIOException fp act =
 getFileContents :: NormalizedFilePath -> Action (FileVersion, Maybe StringBuffer)
 getFileContents = use_ GetFileContents
 
-getFileExists :: NormalizedFilePath -> Action Bool
-getFileExists =
+getFileExists :: VFSHandle -> NormalizedFilePath -> IO Bool
```

Comment:
I’m not sure if/where this is still used. It looks like this should probably become internal to the implementation of the `getFileExists` rules.